vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -383,6 +386,10 @@ func (feeder *clusterStateFeeder) LoadRealTimeMetrics() {
 	for _, containerMetrics := range containersMetrics {
 		for _, sample := range newContainerUsageSamplesWithKey(containerMetrics) {
 			if err := feeder.clusterState.AddSample(sample); err != nil {
+				// Not all pod states are tracked in memory saver mode
```

Comment:
nit: s/pod/container, since the keyError is raised for the container id in the metric

Reply:
I did mean pod state because the `KeyError` will be from this check [here](https://github.com/kubernetes/autoscaler/blob/b232915c2485345d3ae8e587bc3fa933bc5c70d9/vertical-pod-autoscaler/pkg/recommender/model/cluster.go#L180)

Reply:
Ah, you're right, sorry!